pkg/pillar/cmd/zedrouter/acl.go

```diff
@@ -244,11 +246,65 @@ func createACLConfiglet(ctx *zedrouterContext, aclArgs types.AppNetworkACLArgs,
 	if err != nil {
 		return rules, depend, err
 	}
+	clearUDPFlows(aclArgs, ACLs)
 	rules = append(rules, dropRules...)
 	rules, err = applyACLRules(aclArgs, rules)
 	return rules, depend, err
 }
 
+func clearUDPFlows(aclArgs types.AppNetworkACLArgs, ACLs []types.ACE) {
+	var protocol, port string
+	for _, ace := range ACLs {
+		for _, match := range ace.Matches {
+			switch match.Type {
+			case "protocol":
+				protocol = match.Value
+			case "lport":
+				port = match.Value
+			}
+		}
+		if protocol == "" && port != "" {
+			// malformed rule.
+			continue
+		}
+		if protocol != UDPProtocol {
```

Comment:
Don't we allow "any" for protocol? Or is that only in the UI?

Reply:
For port map rules in UI, we only allow TCP, UDP & ICMP